api_cgn_operator_search.yaml

```diff
@@ -3,7 +3,7 @@ info:
   version: 1.0.0
   title: Carta Giovani Nazionale Operator Search API
 host: localhost
-basePath: /api/v1/cgn-operator-search
+basePath: /api/v1/cgn/operator-search
```

Comment:
now the base path is the same with `api_cgn.yaml`. If the client that use these api is the same maybe is better use only one file of specification instead of two.

Reply:
Clients are separated, we prefer to keep them separated to keep also the 2 functions apps domains separated. /cgn points to functions-cgn, /cgn/operator-search points to functions-cgn-operator-search.

Reply:
Ok perfect.